app/src/main/java/com/amaze/filemanager/filesystem/HybridFileParcelable.java

```diff
@@ -154,6 +156,14 @@ public void setPermission(String permission) {
     this.permission = permission;
   }
 
+  public Uri getFullUri() {
+    return fullUri;
+  }
+
+  public void setFullUri(Uri fullUri) {
+    this.fullUri = fullUri;
```

Comment:
This should crash if the full uri is added for anything that is not a DOCUMENT_FILE.